Makefile

```diff
@@ -267,8 +264,8 @@ endif
 
 test-sim-nondeterminism:
 	@echo "Running non-determinism test..."
-	@cd ${CURRENT_DIR}/simapp && go test -mod=readonly -run TestAppStateDeterminism -Enabled=true \
-		-NumBlocks=100 -BlockSize=200 -Commit=true -Period=0 -v -timeout 24h
+	@cd ${CURRENT_DIR}/simapp && go test -mod=readonly -timeout=30m -tags='sims' -run TestAppStateDeterminism \
+		-NumBlocks=100 -BlockSize=200 -Period=0
```

Comment:
Is there a way to keep the flags that are removed and just mark them as deprecated + warn at startup?

We should not use their values, but just warn users about the fact that they are deprecated.  Then we can remove them in some later SDK release